src/Pages/Templates/TemplateDetails/components/AddSystems/AddSystemModal.tsx

```diff
@@ -67,9 +72,14 @@ const useStyles = createUseStyles({
     paddingTop: '0',
     paddingBottom: '0',
   },
+  fullWidth: {
+    width: 'auto',
+    maxWidth: 'unset',
+  },
```

Comment:
there is still some custom CSS in our app, but the goal going forward is to minimize that to or close to zero 🗑️ 
is there no way to achieve something similar (so the filter text doesn't get cut off) as this with some layout components or built-in classes in PF6? could you please look at that 🙏🏼😅

Reply:
sadly i wasn't able to get it working using pf classes